shopfloor_base/utils.py

```diff
@@ -65,3 +68,17 @@ def _get_running_env():
 
 
 RUNNING_ENV = _get_running_env()
+
+
+def _lock(cr, name):
```

Comment:
I would move this to an action. We can have `self._actions_for("lock").advisory(name)`.

Reply:
we can have `for_update` too and replace `_lock_lines` in cluster_picking and location_content_transfer

Reply:
Thanks for the input i wasn't aware of  the actions, this a better place to put it in.
I created the new action and replaced the calls